client/components/info-popover/index.jsx

```diff
@@ -11,84 +11,82 @@ import Popover from 'components/popover';
 import classNames from 'classnames';
 import analytics from 'lib/analytics';
 
-export default React.createClass( {
+export default class InfoPopover extends Component {
+	static propTypes = {
+		autoRtl: PropTypes.bool,
+		className: PropTypes.string,
+		gaEventCategory: PropTypes.string,
+		id: PropTypes.string,
+		ignoreContext: PropTypes.shape( {
+			getDOMNode: PropTypes.func
+		} ),
+		popoverName: PropTypes.string,
+		position: PropTypes.oneOf( [
```

Comment:
@sirreal I'm noticing that there are usages of `position` that don't match these `propTypes`:

![](https://cldup.com/S9JJLuYUiz.png)

To reproduce, go to http://calypso.localhost:3000/plans/$site where $site is a WordPress.com site.